### PR TITLE
Update tab styling for active state and padding

### LINKED ISSUE #146: Tabs highlighting isn't prominent enough for the users to identify which tab is selected.
- [ ] Ideate a few other approaches
- [ ] run some A/B UserZoom tests

### PR DESCRIPTION
**Pull request by taysea**
#### What does this PR do?
Updates tab styling for active state and increases horizontal padding to create better affinity between tab icon + label.

#### What testing has been done on this PR?
Tested locally in aries-site.

#### What are the relevant issues?
Closes #146 
Closes https://github.com/grommet/hpe-design-system/issues/1363
Closes https://github.com/grommet/hpe-design-system/issues/1385

#### Screenshots (if appropriate)
Screen Shot 2021-02-01 at 9 39 45 AM

#### Is this change backward compatible or could it be a breaking change for the official HPE theme?

Introduces new tab styling.

#### How should this PR be communicated in the release notes?
Active tabs have a "medium" width border of border-strong and increased horizontal padding from "small" to "medium".

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -1009,21 +1009,28 @@ export const hpe = deepFreeze({
     },
     border: {
       side: 'bottom',
-      color: 'border',
+      color: 'transparent',
+      size: 'medium',
       active: {
         color: 'border-strong',
       },
       disabled: {
```
Comment (ericsoderberghp) [inline src/js/index.js:1017]: Would it work to do `disabled: undefined` or `disabled: {}` or `disabled: { color: undefined }` instead? Just trying to keep the DOM cleaner where possible.

Comment (taysea) [reply src/js/index.js:1017]: I will double check but I believe we needed transparent to maintain the proper dimensions / text alignment. let me try with undefined now though.

Comment (taysea) [reply src/js/index.js:1017]: I've done `disabled: { color: undefined }` in the site locally, and that works.

Comment (taysea) [reply src/js/index.js:1017]: same for hover.

**Review by ericsoderberghp — APPROVED** [review]

**Pull request merged**